store.go

```diff
@@ -16,6 +16,7 @@ package gitstore
 import (
 	"flag"
 	"fmt"
+	"path"
```

Comment:
the `path` package deals with HTTP paths, rather than file paths which is what we're dealing with here. We should use `"path/filepath"` instead (although I suspect we don't have any users on windows anyway :P ).

Reply:
Good point, @theobarberbany Can you fix this and the other reference up please and check everything is still ok?